docroot/themes/humsci/humsci_basic/src/js/shared/vertical-tabs/vertical-tabs.js

```diff
@@ -0,0 +1,10 @@
+function closeDetails() {
+  // Close Revision Information Details element by default.
+  if (document.getElementById('edit-revision-information')) {
```

Comment:
Is it possible to use a more specific selector? There can be other element with the same ID in a different form and we need to make sure that we're targeting the right one.